index.html

```diff
@@ -1,162 +1,250 @@
 <!DOCTYPE html>
-    <html lang="en">
-        <head>
```

Comment:
Please do not mess up with the indentation of the parts of the code you are not working on